docker/install/ubuntu_install_qemu.sh

```diff
@@ -45,7 +51,7 @@ gpg --verify qemu-5.1.0.tar.xz.sig
 tar -xf qemu-5.1.0.tar.xz
 cd qemu-5.1.0
 ./configure --target-list=aarch64-softmmu,arm-softmmu,i386-softmmu,riscv32-softmmu,riscv64-softmmu,x86_64-softmmu
-make -j2
+make -j${NUM_CORE}
```

Comment:
Why not use `nproc`?

Reply:
Because some people are building this on a local machine to test with low number of cores and it could crash.